### ISSUE TITLE
Disable VHACD in urdf conversion

### ISSUE THREAD

**Issue by richardrl** [mikedh/trimesh#1277]
Would be really nice to turn off the VHACD. VHACD messes up meshes all the time (makes planar faces non planar), we should have the option to disable for convex pieces (or detect convexity automatically).

**Comment by mikedh** [comment]
Yeah it would be very straightforward to add a flag for that (or check `mesh.is_convex`), PR's welcome!

The trimesh URDF exporter is pretty simplistic, I looked into adding [support for kinematics](https://github.com/mikedh/kinematic/) but wasn't sure if trimesh was the right place for that, and it would be a ton of work to get the importers/exporters to be actually lossless and schema-following to enable [something like this](https://gist.github.com/mikedh/ce09d1bf83767fefded25be519a61564).  

There are packages that operate on top of trimesh like [urdfpy](https://github.com/mmatl/urdfpy) that do a nice job being format-specific fwiw.
